tests/platform_tests/api/test_sfp.py

```diff
@@ -97,27 +97,32 @@ class TestSfpApi(PlatformApiTestBase):
     chassis_facts = None
     duthost_vars = None
     num_sfps = None
-    present_sfp = None
+    candidate_sfp = None
 
     # This fixture would probably be better scoped at the class level, but
     # it relies on the platform_api_conn fixture, which is scoped at the function
     # level, so we must do the same here to prevent a scope mismatch.
     @pytest.fixture(scope="function", autouse=True)
-    def setup(self, platform_api_conn):
+    def setup(self, request, platform_api_conn):
+        self.skip_absent_sfp = request.config.getoption("--skip_absent_sfp")
```

Comment:
Suggest using hyphens in lieu of underscores in argument name (e.g., `--skip-absent-sfp`)